experiments.py

```diff
@@ -1,362 +1,139 @@
-    model = trainRL(model_generator=model_generator,
-                    train_timesteps=TRAIN_TIMESTEPS, 
-                    board_height=BOARD_HEIGHT, 
-                    board_width=BOARD_WIDTH, 
-                    visualization_fps=VIS_FPS, 
-                    reward_function=RewardFuncs.reward_2x_closer_to_fruit)
-    scores = testRL(model=model, 
-                    test_timesteps=TEST_TIMESTEPS, 
-                    board_height=BOARD_HEIGHT, 
-                    board_width=BOARD_WIDTH, 
-                    visualize_testing=VISUALIZE_TESTING, 
-                    visualization_fps=VIS_FPS, 
-                    reward_function=RewardFuncs.reward_2x_closer_to_fruit)
-    analyze_and_write_to_csv(strategy_label, strategy_description, scores)
-
-    # Distance Reward Structure with 10x Reward
-    strategy_label = "("+model_type+"): "+"Reward Closer to Fruit with 10x Reward"
-    strategy_description = "Very similar to the basic reward structure, but we reward ten times as much as punish when the snake moves a step closer to the fruit"
-    model = trainRL(model_generator=model_generator,
-                    train_timesteps=TRAIN_TIMESTEPS, 
-                    board_height=BOARD_HEIGHT, 
-                    board_width=BOARD_WIDTH, 
-                    visualization_fps=VIS_FPS, 
-                    reward_function=RewardFuncs.reward_10x_closer_to_fruit)
-    scores = testRL(model=model, 
-                    test_timesteps=TEST_TIMESTEPS, 
-                    board_height=BOARD_HEIGHT, 
-                    board_width=BOARD_WIDTH, 
-                    visualize_testing=VISUALIZE_TESTING, 
-                    visualization_fps=VIS_FPS, 
-                    reward_function=RewardFuncs.reward_10x_closer_to_fruit)
-    analyze_and_write_to_csv(strategy_label, strategy_description, scores)
-
     # Kill after 10 idle moves
     strategy_label = "("+model_type+"): "+"Kill after 10 idle moves"
     strategy_description = "Here we just do the basic reward structure of + for fruit and - for wall. We kill the snake after a set number of idle moves."
-    model = trainRL(model_generator=model_generator,
-                    train_timesteps=TRAIN_TIMESTEPS, 
-                    board_height=BOARD_HEIGHT, 
-                    board_width=BOARD_WIDTH,
-                    max_moves_no_fruit=10,
-                    visualization_fps=VIS_FPS, 
-                    reward_function=RewardFuncs.basic_reward_func)
-    scores = testRL(model=model, 
-                    test_timesteps=TEST_TIMESTEPS, 
-                    board_height=BOARD_HEIGHT, 
-                    board_width=BOARD_WIDTH, 
-                    max_moves_no_fruit=10,
-                    visualize_testing=VISUALIZE_TESTING, 
-                    visualization_fps=VIS_FPS, 
-                    reward_function=RewardFuncs.basic_reward_func)
+    scores = train_and_testRL(model_generator, RewardFuncs.basic_reward_func, model_filename=strategy_label)
```

Comment:
nit: comment above about killing after 10 moves doesn't match the basic reward function we're using